pom.xml

```diff
@@ -957,6 +957,9 @@
               <systemId>src/main/resources/com/puppycrawl/tools/checkstyle/configuration_1_3.dtd</systemId>
             </validationSet>
           </validationSets>
+          <catalogs>
+            <catalog>config/catalog.xml</catalog>
+          </catalogs>
```

Comment:
@romani 
It is the config for Catalog Files feature of maven xml plugin.
See http://www.mojohaus.org/xml-maven-plugin/examples/catalog.html